actioncable/lib/action_cable/channel/base.rb

```diff
@@ -259,6 +261,8 @@ def dispatch_action(action, data)
           else
             public_send action
           end
+          rescue Exception => exception
+            rescue_with_handler(exception) || raise
```

Comment:
This indentation is off... and we should probably add a test for the new behaviour too.